File: fix.py
import sys

path = './src/components/StickerDrag.tsx'
with open(path, 'r') as f:
    text = f.read()

text = text.replace('\\${', '${')
text = text.replace('\\`', '`')

with open(path, 'w') as f:
    f.write(text)
print("Done fixing file")

File: fix.js
const fs = require('fs');
const file = './src/components/StickerDrag.tsx';
let content = fs.readFileSync(file, 'utf8');

// The file currently has literal \ followed by $ (e.g. \${canvasWidth})
content = content.replace(/\\\$/g, '$');
// It also has \ followed by ` 
content = content.replace(/\\`/g, '`');

fs.writeFileSync(file, content);
console.log('Fixed escaping!');
